src/layout-forceatlas2/README.md

```diff
@@ -31,7 +31,7 @@ Note also that the algorithm has an edge-case where the layout cannot be compute
 - **adjustSizes** _?boolean_ [`false`]: should the node's sizes be taken into account?
 - **barnesHutOptimize** _?boolean_ [`false`]: whether to use the Barnes-Hut approximation to compute repulsion in `O(n*log(n))` rather than default `O(n^2)`, `n` being the number of nodes.
 - **barnesHutTheta** _?number_ [`0.5`]: Barnes-Hut approximation theta parameter.
-- **edgeWeightInfluence** _?number_ [`0`]: influence of the edge's weights on the layout.
+- **edgeWeightInfluence** _?number_ [`0`]: influence of the edge's weights on the layout. The `weight` attributes of edges is used as the weight of each edge.
```

Comment:
I think you mentioned the name of the `weight` attribute was customizable by an option. Is that actually true?

Reply:
I said that it is customizable everywhere else in the various `graphology` libraries yes. But it's not the case with FA2, which feels inconsistent. I have open the issue #269 to fix that in a near future.